bundle/Resources/config/kernel.yml

```diff
@@ -2,3 +2,9 @@ system:
     default:
         fielddefinition_edit_templates:
             - { template: "EzSystemsRepositoryFormsBundle:ContentType:field_types.html.twig", priority: 0 }
+        location_view:
+            content_name:
+                landing_page:
+                    template: "EzSystemsRepositoryFormsBundle:content_name:content_name.html.twig"
+                    match:
+                        Identifier\ContentType: [landing_page,folder]
```

Comment:
For information, matching is done here: https://github.com/ezsystems/ezpublish-kernel/blob/master/eZ/Publish/Core/MVC/Symfony/Matcher/AbstractMatcherFactory.php#L103